src/note/note.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable, Logger } from '@nestjs/common';
+import {InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+
+import { notes } from './note.entity';
+import { Note } from '../interfaces/note'
+@Injectable()
+export class  NoteService {
+constructor(
+@InjectRepository(notes)
+private noteRepository: Repository<notes>,
+) {}
+async index(): Promise<Note[]> {
+return await this.noteRepository.find();
+}
+async findOne(id): Promise<Note[]>{
+return await this.noteRepository.find({where: [{ 'id': id }]});
```

Comment:
`this.noteRepository.find({where: [{ 'id': id }]});`

można by uprościć do 
`this.noteRepository.findOne(id);`

Reply:
Poprawione Później robiłem refaktor tej metody i zastosowałem findOneOrFail NIe wiem czy to ok , pozostałość z larawela tam była podobna :)